.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+
+### Creating pull requests
+
+Changes to the model _cannot_ be directly committed to the `master` or `devel` branches (in fact they are protected). Commits are made to side-branches, after which pull requests are made for merging with `devel`. For this, follow the [instructions](#contributing-to-the-model) for contributors, but consider that as members of the development team have write access to the repository, you can create a branch directly in the main repository without needing to fork, for your convenience. This means that you can:
+
+* Skip step 2 of the contribution process.
+* In step 3 of the contribution process, clone directly the original repo:
+  ```
+  git clone https://github.com/SysBioChalmers/yeast-GEM.git
+  ```
+
+Follow all other steps in the same way. Also, when creating your pull request (or after you have created it):
+* Choose 1 or more members of the team (ideally with knowledge on the pull request) as reviewers. Note that the person making the pull request and the reviewer _cannot_ be the same person.
+* Assign appropriate [labels](https://github.com/SysBioChalmers/yeast-GEM/issues/labels).
+* Assign the pull request to one of the available [projects](https://github.com/SysBioChalmers/yeast-GEM/projects), if applicable.
+	
+### Reviewing pull requests
+
+Every pull request must be approved by at least one reviewer before it can be merged. When reviewing someone else's pull request, keep in mind the following aspects:
+* **Compatibility:** First of all, make sure that the model is still compatible with the loading/saving wrappers (`loadYeastModel.m` & `saveYeastModel.m`) and that no errors appear. Check also that [`dependencies.txt`](https://github.com/SysBioChalmers/yeast-GEM/blob/master/ModelFiles/dependencies.txt) does not change in any unexpected ways (e.g. an "unknown" toolbox version). Finally, ensure that the SBML fields `model metaid`, `model id` and `model name` never change, as if they change it would create a conflict in the next release.
+* **Documentation:** Every change should be justified with a reference/link/argument. This can be provided as data in `/ComplementaryData`, or directly as a comment in the pull request.
+* **Reproducibility:** If there are any added scripts, make sure that if you run them, the model gets updated from how it was in `devel` to how it is in the pull request. For this, you may _locally_ switch to the corresponding branch, replace the `.xml` file with the one from `devel` before the changes, and run the associated scripts. Remember to stash any undesired changes afterwards.
+* **Style:** Ensure that the changes to the model are compliant with the model's rxn/met/gene naming conventions (when unsure, take a look at a similar field in the model). Also, make sure that scripts have a compliant style, and datasets are straight-forward to understand.
```

Comment:
Should the naming convention (especially IDs) be specified somewhere? Can IDs be reused for instance?

Reply:
It should definitively be specified. However, I think that requires a full section and would rather take it on a new PR with more discussion. In the meantime, I have included a mention of `getNewIndex.m` for generating new met/rxn ids in step 6 of "Contributing to the model" (and I will soon work on documenting said function).